redaxo/src/core/lib/response.php

```diff
@@ -62,6 +62,8 @@ public static function sendRedirect($url)
             throw new InvalidArgumentException('Illegal redirect url "' . $url . '", contains newlines');
         }
 
+        self::cleanOutputBuffers();
```

Comment:
Es gibt einige callsites von sendRedirect() wo man jetzt das output-buffer löschen weglassen kann

Reply:
Was lernen wir aus der Geschichte: Nicht PR-fremde Dinge mit in einen PR packen. :D
Hätte ich mal nicht angefangen mit `::cleanOutputBuffers`.. ;) 

Ok, ich schaue mal.